### PR TITLE
Docs: Testsuite: Grammar

### PR DESCRIPTION
**Pull request by Jonta**

### CODE REVIEW

File: docs/testsuite [hunk 1]
```diff
@@ -4,7 +4,7 @@ Michael Stapelberg <michael@i3wm.org>
 September 2012
 
 This document explains how the i3 testsuite works, how to use it and extend it.
-It is targeted at developers who not necessarily have been doing testing before
+It is targeted at developers who have not necessarily been doing testing before
```
Comment (orestisfl) [inline docs/testsuite]: I think present perfect would be even simpler here: "who haven't necessarily done testing before or..."

**Review by orestisfl — CHANGES_REQUESTED** [review]
Thanks ;)

File: docs/testsuite (hunk 1 above)
Comment (Jonta) [reply docs/testsuite]: I agree

Comment (orestisfl) [reply docs/testsuite]: So, do you want to update your PR? :)

Comment (Jonta) [reply docs/testsuite]: @orestisfl Done

**Review by orestisfl — APPROVED** [review]

**Pull request merged**